pyiceberg/catalog/glue.py

```diff
@@ -105,6 +105,10 @@
 GLUE_SKIP_ARCHIVE = "glue.skip-archive"
 GLUE_SKIP_ARCHIVE_DEFAULT = True
 
+# Configure an alternative endpoint of the Glue service for GlueCatalog to access.
+# This could be used to use GlueCatalog with any glue-compatible metastore service that has a different endpoint
+GLUE_CATALOG_ENDPOINT = "glue.endpoint"
```

Comment:
Looks good, same property as in Java: https://github.com/apache/iceberg/blob/f614a3f8b0379188efcc0c38caa486c94c6a52a3/aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java#L103